code/modules/mailorder/modr_entries.dm

```diff
@@ -0,0 +1,365 @@
+		desc = "Prepared with one of our signature flavors - try them all!"
+		order_items = list(/obj/item/reagent_containers/food/snacks/soup/oatmeal)
+		cost = PAY_TRADESMAN / 5
+
+	coconut
+		name = "Coconut"
+		desc = "Delicious and versatile, bringing you a burst of coco-fun."
+		order_items = list(/obj/item/reagent_containers/food/snacks/plant/coconut)
+		cost = PAY_TRADESMAN / 3
+
+	cucumber
+		name = "Cucumber"
+		desc = "A delicate vegetable hand-picked for minimum bitterness."
+		order_items = list(/obj/item/reagent_containers/food/snacks/plant/cucumber)
+		cost = PAY_TRADESMAN / 4
+
+	pineapple
+		name = "Pineapple"
+		desc = "Rich tropical flavor in a spiky package, express-delivered."
+		order_items = list(/obj/item/reagent_containers/food/snacks/plant/pineapple)
+		cost = PAY_TRADESMAN / 3
+
+	pumpkin
+		name = "Pumpkin"
+		desc = "Shipped whole and intact - perfect for autumnal classics."
+		order_items = list(/obj/item/reagent_containers/food/snacks/plant/pumpkin)
+		cost = PAY_TRADESMAN / 4
+
+	catnip
+		name = "Catnip Sprig"
+		desc = "Pure nepeta cataria, perfect for your feline pals."
+		order_items = list(/obj/item/plant/herb/catnip)
+		cost = PAY_TRADESMAN / 5
+
+	mint
+		name = "Mint Sprig"
+		desc = "Effervescent mint, carefully preserved for peak flavor."
+		order_items = list(/obj/item/plant/herb/mint)
+		cost = PAY_TRADESMAN / 5
+
+	cereal_pope
+		name = "Pope Crunch Cereal"
+		desc = "Made with only the highest-quality iron supplements."
+		order_items = list(/obj/item/reagent_containers/food/snacks/cereal_box)
+		cost = PAY_UNTRAINED / 3
+
+	cereal_wonks
+		name = "Honey Wonks Cereal"
+		desc = "Your sticky-sweet sugar rush, at a super special price."
+		order_items = list(/obj/item/reagent_containers/food/snacks/cereal_box/honey)
+		cost = PAY_UNTRAINED / 3
+
+	cereal_monki
+		name = "Tanh-O-Nys Cereal"
+		desc = "Does it taste like banana? Who knows! Buy some and find out."
+		order_items = list(/obj/item/reagent_containers/food/snacks/cereal_box/tanhony)
+		cost = PAY_UNTRAINED / 3
+
+	cereal_roach
+		name = "Roach Puffs Cereal"
+		desc = "Everybody's favorite chocolate-themed roach-themed treat."
+		order_items = list(/obj/item/reagent_containers/food/snacks/cereal_box/roach)
+		cost = PAY_UNTRAINED / 3
+
+	cereal_bundle
+		name = "Munch 'Em All Bundle"
+		desc = "Can't decide on a cereal? Buy them all and save!"
+		order_items = list(/obj/item/reagent_containers/food/snacks/cereal_box,
+		/obj/item/reagent_containers/food/snacks/cereal_box/honey,
+		/obj/item/reagent_containers/food/snacks/cereal_box/tanhony,
+		/obj/item/reagent_containers/food/snacks/cereal_box/roach)
+		cost = PAY_UNTRAINED
+
+
+//Survival Mart: primarily medical, but not exclusively
+ABSTRACT_TYPE(/datum/mail_order/survmart)
+/datum/mail_order/survmart
+
+	oxytank
+		name = "Emergency Oxygen Tank"
+		desc = "An easy-to-use compact gas tank pre-loaded with pure oxygen."
+		order_items = list(/obj/item/tank/emergency_oxygen)
+		cost = PAY_TRADESMAN
```

Comment:
Pretty neutral to this, but given how much an emergency O2 tank mitigates depressurization I think this might trivialize things too much.

Reply:
Idea comes to mind: post-introduction (removing this in initial addition), create and add a compact gas tank that doesn't come with gasses, requiring either filling it with air mix from one of the publicly accessible tanks (cheap but not as trivial to keep your air supply up) or paying for / having access to oxygen.